music/src/gansynth/index.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+export {GANSynth} from './model';
```

Comment:
Could you also export `specgramsToAudio` so users can form the audio buffer more easily?

Like

```ts
export { specgramsToAudio } from "./audio_utils";
```

Or expose that functionality as a method in the `GANSynth` class.

Reply:
Sure thing, I'll expose it as a class.